src/nxdoc/nuxeo-server/authentication-and-user-management/how-to-add-new-fields-to-the-user-profile-or-group-profile.md

```diff
@@ -72,7 +73,9 @@ In the Nuxeo Platform **user** infos are displayed as follow:
 
 {{#> callout type='note'}}
 Whenever you contribute to a schema referencing another schema or document, you **must** define the references and inverse references. On that particular case you also have to define **directories** to specify the way schemas are linked to each others.
-The relation between `user` and `groups` uses a directory called **groupDirectory** defined as follow:
+The relation between `user` and `groups` uses a directory called **groupDirectory** defined below:
+{{/callout}}
```

Comment:
@manonlumeau I've moved the codeblock out of the callout because the codeblock causes issues with the callout.